src/helpers.js

```diff
@@ -8,17 +11,17 @@ export const isElementAnSFC = (element) => {
   return !element.type.prototype.isReactComponent;
 };
 
-export function omit(obj, attrs = []) {
-  const result = {};
-  Object.keys(obj).forEach((key) => {
+export function omit<R: {}, T: R>(obj: T, attrs: $Keys<T>[] = []): R {
+  const result: $Shape<T> = {};
+  Object.keys(obj).forEach((key: $Keys<T>) => {
```

Comment:
The second jeneric lets you specify exact type you want to return: `const B: Type = omit(A, ...keys)`